### ISSUE TITLE
Documentation? Video demonstration?

### ISSUE THREAD

**Issue by j** [josuelopezv/aDrums#1]
Are you still using this? And if so, do you have a demo online?

**Comment by josuelopezv** [comment]
Hi,


Yes this is working but it was still under development but anything you
need i can help you

**Comment by j** [comment]
It'd be cool to have direct links of what exactly all is needed as well as instalation of this library.  I have a new custom accoustic kit being made right now and going to order r-drum triggers and jobeky cymbals, etc, but would like to hack away at this using my pearl e-Pro just to get a feel.

What hardware do I need exactly?

Such as, what is "rest of electronics".  What PCB board should I get, etc?

Thanks!

**Comment by j** [comment]
Also, have you tried this with something like Kontakt 5?  The Jay Maas signature series sampling from RoomSound is so nice. https://www.youtube.com/watch?v=6PCLWqnZ0bA

**Comment by alensiljak** [comment]
Yes, please! Some documentation in the wiki would be fantastic. I'll try to work out some on my own, of course, and contribute via fork.
Other than that, I'd like to show you a link to [Open Source Drum Module](https://sites.google.com/view/open-source-drum-module) project. Basicall, it's a site where I'm trying to collect various pieces, work out some standards, and present multiple end-options by combining various components together. OK, this sounds too abstract but practically means that one should be able to have an open source drum module by using their PC for home practice, as well as having a separate enclosed module for professional use.
Well, have a look at the text and let me know where it can be improved. I could also initially help by writing some documentation from the point of a newcomer to aDrum, which usually helps.
I clearly see that aDrum idea fully aligns with OSDM.

**Comment by josuelopezv** [comment]
Hi All,

thanks for the interest in my project. I started this project because i had
the need of something like mega drums (http://www.megadrum.info) so I
created an open source version of it. I have not had the chance to finish
the docs or upload any demo although most of the project was fully
functional I was waiting to finish it if someone wants to help me that will
be great. I finish the hardware and firmware and only missing part is 30%
of the development of the software to upload the config files to the
hardware.

**Comment by alensiljak** [comment]
Fantastic! I'm certainly volunteering to help. I can't promise a whole lot of time but I can say that we definitely share exactly the same goal.
There are a few others who made some progress on the hardware part and this can be followed [here](https://www.vdrums.com/forum/advanced/diy/1154514-thinking-of-building-my-own-module).
I see that the software was built with Visual Studio so that should not be a problem.

Do you have any schematics you could share? I'd like to eventually have a prototype with a breadboard and Arduino Mega, that I could use to test everything.
I could immediately focus on the missing part - the upload of the config files to the hardware. It's just that I'd need to test with actual hardware. :)

**Comment by j** [comment]
Food for thought: keeping the software OS but having a paid hardware.. I'd buy that over building it myself and just hack on software side.  Easier way to get into the code.  I just bought the Pearl Mimic Pro though...

**Comment by josuelopezv** [comment]
the schematics were uploaded on the project as well. it is a simple arduino
with multiplexer to expand the inputs

**Comment by alensiljak** [comment]
Hi, @josuelopezv, could you post a link to the schematics, please? I see no new commits so they must be elsewhere? Looking forward to hack a bit with this.
Thanks!

**Comment by josuelopezv** [comment]
https://github.com/josuelopezv/aDrums/blob/master/Eagle/aDrums/drumuino.sch
